includes/class-simple-local-avatars.php

```diff
@@ -684,7 +687,12 @@ public function admin_enqueue_scripts( $hook_suffix ) {
 		 * @param string $hook_suffix Page hook.
 		 * @return array
 		 */
-		$screens = apply_filters( 'simple_local_avatars_admin_enqueue_scripts', array( 'profile.php', 'user-edit.php', 'options-discussion.php' ), $hook_suffix );
```

Comment:
thanks for bringing this to my attention, I have just noticed the `$hook_suffix` is a parameter coming with the [admin_enqueue_scripts](https://developer.wordpress.org/reference/hooks/admin_enqueue_scripts/) only, not with [wp_enqueue_scripts](https://developer.wordpress.org/reference/hooks/wp_enqueue_scripts/). So we can keep the filter as is because it will only be applicable to the BE side. Probably we can introduce a new filter to load JS on specific pages at FE in the future. For now, reverting it back.